gravitee-apim-console-webui/src/management/settings/api-portal-header/migrated/header-dialog/header-dialog.component.html

```diff
@@ -0,0 +1,49 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+
+<h4 mat-dialog-title>{{ isUpdate ? "Update" : "New"}} API Header</h4>
+<div mat-dialog-content class="controls">
+  <mat-form-field class="control">
+    <mat-label>Name</mat-label>
+    <input required matInput [(ngModel)]="data.name" data-testid="name-input">
+  </mat-form-field>
+  <mat-form-field class="control">
+    <mat-label>Value</mat-label>
+    <input required matInput [(ngModel)]="data.value" data-testid="value-input">
+  </mat-form-field>
+</div>
+<div mat-dialog-actions class="action-buttons">
+  <button
+    mat-raised-button
```

Comment:
```suggestion
    mat-flat-button
```
If you look at the design of the dialog migrate normally it's `falt` for cancel and `raised` for main action only